pmg/templates/blog_post.html

```diff
@@ -8,8 +8,13 @@
   <div class="row">
     <div class="col-sm-8">
       <div class="single-post">
+        <div class="date">{{ post.date.strftime('%d %B %Y') }}</div>
         <h2 class="title">{{ post.title }}</h2>
-        <div class="date">{{ post.date.strftime('%d-%m-%Y') }}</div>
+        <div class="preview-image">
+            {% if post.get_preview_image() %}
+              <img src="{{ post.get_preview_image().url }}" alt="{{ post.get_preview_image().title }}" />
```

Comment:
It feels wrong calling `get_preview_image` three times, but I'm not sure what else to do here?

I could return the image url (or the placeholder url if the image does not exist) from `get_preview_image`, but then we'll lose the image's `title` property.

Reply:
there might be "assign" semantics in jinja but let's see how this scales and improve on it if really necessary. It's not like it's going to hit the DB three times (I'd assume)

Reply:
Ah looks like there is indeed an "assign" semantic. Will try it out!